### ISSUE TITLE
Table with sortable headers: JAWS cant read cell contents

### ISSUE THREAD

**Issue by mae-financial-com** [bootstrap-vue/bootstrap-vue#6637]
### Describe the bug

We have JAWS screen reader running. In a bootstrap table which has sortable headers if we 'tab' to the last sortable header and then press down arrow key the screen reader should be able to read the first cell in the table. The contents of the table cant be read instead the page scrolls down.

### Steps to reproduce the bug

1. Go to https://bootstrap-vue.org/docs/components/table#sorting
2. Turn on screen reader(JAWS)
3. Tab to the last sortable header of the table and then press down arrow to read the cells of the table.
4. The cell values are not read instead the page gets scrolled.

### Expected behavior

JAWS should be able to read all the contents of the table using down arrow keys. Each cell contents should be read one by one.

### Versions

**Libraries:**

- BootstrapVue: 2.13.1
- Bootstrap: 4.4.1
- Vue: 2.6.12

**Environment:**

- Windows Laptop
- OS: Windows 10
- Browser: All browsers
- Version: [e.g. 70]

### Demo link

Table in this section
https://bootstrap-vue.org/docs/components/table#sorting
